pkg/apis/core/validation/validation.go

```diff
@@ -2957,6 +2959,15 @@ func validateExecAction(exec *core.ExecAction, fldPath *field.Path) field.ErrorL
 	return allErrors
 }
 
+func validateSleepAction(sleep *core.SleepAction, gracePeriod int64, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	if sleep.Seconds <= 0 || sleep.Seconds > gracePeriod {
+		invalidStr := fmt.Sprintf("must be greater than 0 and may not be longer than `terminationGracePeriodSeconds` (%d)", gracePeriod)
```

Comment:
nit:  "must be greater than 0 and less than `terminationGracePeriodSeconds` (%d)"